tests/cache/models/redisTestUtil.js

```diff
@@ -21,6 +21,54 @@ const Sample = tu.db.Sample;
 const testStartTime = new Date();
 
 module.exports = {
+  populateRedisWithoutSample(done) {
+    let a1;
```

Comment:
Do you want to do the tests without using populateRedisWithoutSample? Like what we talked about some time back.